indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -233,6 +245,12 @@ public TaskStatus call()
                               command.add(String.format("-Ddruid.host=%s", childHost));
                               command.add(String.format("-Ddruid.port=%d", childPort));
 
+                              if(config.isSeparateIngestionEndPoint()) {
+                                command.add(String.format("-Ddruid.indexer.task.chathandler.service=%s", "druid/chathandler"));
```

Comment:
These three properties are used to create `DruidNode` object annotated with `RemoteChatHandler` at `ServiceAnnouncingChatHandlerProvider` and `getServer` method of `ChatHandlerServerModule`. However, while announcement of the event receiver firehose the service name provided during registration is used and the  'serviceName' property which is specified here is ignored. Thus, it is not consumed anywhere, it is just there so that object creation does not fail.